### ISSUE TITLE
JulesJordan Problem

### ISSUE THREAD

**Issue by pibi73** [PAhelper/PhoenixAdult.bundle#1906]
Hi Guys,

anyone have problem with JulesJordan Scenes?
i can't index any .

Thanks

**Comment by flameknuckle** [comment]
yeah i wasn't able to match anything using the official site so i used data18 instead. for example [https://www.data18.com/scenes/3126507](url) would be "data18scenes - 3126507"
